src/admin-program-list/program-administration-list.controller.js

```diff
@@ -44,7 +44,11 @@
          * @description
          * Holds list of all programs with templates.
          */
-        vm.programs = programList;
+        vm.programs = programList.sort((a, b) => {
```

Comment:
Please check if we can use AngularJS approach: https://docs.angularjs.org/api/ng/filter/orderBy#examples instead of creating sorting function